modules/nf-core/propr/logratio/main.nf

```diff
@@ -0,0 +1,23 @@
+process PROPR_LOGRATIO {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "conda-forge::r-propr=4.2.6"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/r-propr:4.2.6':
+        'biocontainers/r-propr:4.2.6' }"
+
+    input:
+    tuple val(meta), path(count), val(transformation), val(reference), val(alpha)
```

Comment:
You should separate the mandatory configuration options in the the vals from the files to which the `meta` pertains- see [this example](https://github.com/nf-core/modules/blob/master/modules/nf-core/salmon/quant/main.nf) and [the docs](https://nf-co.re/docs/contributing/modules).